ports/quake.2/Quake 2.sh

```diff
@@ -36,21 +42,18 @@ else
   sscale="3"
 fi
 
-# if [[ $param_device == "anbernic" ]]; then
-#   if [[ -f $GAMEDIR/temp/quake2anbernic ]]; then
-#     cp -rf $GAMEDIR/temp/quake2anbernic/* $GAMEDIR/
-#   fi
-# fi
-# 
-# if [ -f $GAMEDIR/temp/quake2anbernic ]; then
-#   rm -rf $GAMEDIR/temp/
-# fi
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
 
 cd $GAMEDIR
 $ESUDO chmod 666 /dev/tty1
 $ESUDO chmod 666 /dev/uinput
-$GPTOKEYB "quake2" -c "$GAMEDIR/quake2.gptk" &
-SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig" ./quake2 +set vid_renderer soft +set r_customwidth $swidth +set r_customheight $sheight +set r_hudscale $sscale +set r_menuscale $sscale -datadir $GAMEDIR  +set vid_fullscreen 2 -portable 2>&1 | tee $GAMEDIR/log.txt
+# $GPTOKEYB "quake2" -c "$GAMEDIR/quake2.gptk" &
+$GPTOKEYB "quake2" &
+SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig" ./quake2 +set vid_renderer gles3 +set r_customwidth $swidth +set r_customheight $sheight +set r_hudscale $sscale +set r_menuscale $sscale -datadir $GAMEDIR  +set vid_fullscreen 2 -portable 2>&1 | tee $GAMEDIR/log.txt
```

Comment:
Adding gles3 as an option might have impact on all other cfws. Has to be tested.